core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -180,8 +192,8 @@ public synchronized void connect() throws AlluxioStatusException {
       try {
         mAddress = getAddress();
       } catch (UnavailableException e) {
-        LOG.warn("Failed to determine master RPC address ({}), retrying: {}",
-            retryPolicy.getAttemptCount(), e.toString());
+        LOG.warn("Failed to determine {} rpc address ({}), retrying: {}",
```

Comment:
technically that isn't what it means since we might fail for different reasons each time, but the retry count keeps going up. Previously we have used
```java
        LOG.warn("Failed to connect ({}) with {} @ {}: {}", retryPolicy.getAttemptCount(),
            getServiceName(), mAddress, e.getMessage());
```
so this is following convention